walkthroughs/howto-k8s-multiple-listeners/manifest.yaml

```diff
@@ -0,0 +1,261 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: colors
+  labels:
+    mesh: colors
+    gateway: colors-gw
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: colors
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: colors
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: colors-gw
+  namespace: colors
+spec:
+  namespaceSelector:
+    matchLabels:
+      gateway: colors-gw
+  podSelector:
+    matchLabels:
+      app: colors-gw
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+    - portMapping:
+        port: 8090
+        protocol: http
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: GatewayRoute
+metadata:
+  name: gateway-route-main
+  namespace: colors
+spec:
+  httpRoute:
+    match:
+      prefix: /
+      port: 8090
+    action:
+      target:
+        virtualService:
+          virtualServiceRef:
+            name: color-headers
+        port: 8090
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: GatewayRoute
+metadata:
+  name: gateway-route-misc
+  namespace: colors
+spec:
+  httpRoute:
+    match:
+      prefix: /
+      port: 8080
+    action:
+      target:
+        virtualService:
+          virtualServiceRef:
+            name: color-headers
+        port: 8080
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: white
```

Comment:
this name is kinda confusing with the routing later, it would maybe be a little less confusing with a name like multi-color or something, since the routes are black and white.

Reply:
yeah i agree-- will change